### DISCUSSION TITLE
Resetdata is not possible

### DISCUSSION THREAD

**Discussion by plemens** [stephanrauh/ngx-extended-pdf-viewer#1205] [Q&A]
Hi, if I reset all the data from the formData in a function and then click in an InputFielt from the PDF in the browser, then I see the last data again.
But the formData object is empty.
All caches are disabled?

What is the problem?

**Comment by stephanrauh** [comment]
I assume you're resetting the `formData` in Angular (as opposed to resetting it in the PDF form)?

You mention the `formData` object is empty. Probably that's the clue. ngx-extended-pdf-viewer iterates over the attributes of `formData` to synchronize with the PDF file. However, if there's an empty object, there's no attribute to iterate. Try setting every attribute to `null` or to empty strings. I suppose you can also use `undefined`, but I'm not sure.
